packages/react/src/components/timeInput/TimeInput.tsx

```diff
@@ -0,0 +1,330 @@
+import React, { FocusEventHandler, useLayoutEffect, useRef, useState } from 'react';
+import 'hds-core';
+import isFunction from 'lodash.isfunction';
+
+import { InputWrapper } from '../../internal/input-wrapper/InputWrapper';
+import { TextInputProps } from '../textInput/TextInput';
+import textInputStyles from '../textInput/TextInput.module.css';
+import styles from './TimeInput.module.scss';
+import classNames from '../../utils/classNames';
+
+export type TimeInputProps = TextInputProps & {
+  /**
+   * A visually hidden label for the hours. Helps to navigate the component with screen readers.
+   */
+  hoursLabel: string;
```

Comment:
Should we make this not compulsory and default to some value?

Reply:
Yes, it is possible but then it would probably require language attribute 🤔